docs/windows.md

```diff
@@ -1,142 +1,139 @@
-cd %GOPATH%/src/github.com/ipfs/go-ipfs
-gx --verbose install --global
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
 ```
+This can be acheived with the `setx` command
+```
+SETX PATH %PATH%;\msys64\usr\bin
+```
+
+## Cygwin
+1. Install Cygwin (https://www.cygwin.com)  
+2. During the install, select the following packages. (If you already have Cygwin installed, run the setup file again to install additional packages) A fresh install should look something like [this reference image](https://ipfs.io/ipfs/QmaYFSQa4iHDafcebiKjm1WwuKhosoXr45HPpfaeMbCRpb/cygwin%20-%20install.png).
+3. 
+    - devel packages
+        - `git`
+        - `make`
+    - archive packages
+        - `unzip`
+    - net packages
+        - `curl`  
+3. Run the following inside a normal `cmd` prompt (Not the Cygwin prompt, we only need Cygwin's tools)  
+An explanation of this block is below.
+```
+SET PATH=%PATH%;\cygwin64\bin
+mkdir %GOPATH%\src\github.com\ipfs
+cd %GOPATH%\src\github.com\ipfs
+git clone https://github.com/ipfs/go-ipfs.git
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
+%GOPATH%\bin\ipfs.exe version --all
+```
+|      |      |
+| ---: | :--- |
+|`SET PATH=%PATH%;\cygwin64\bin`           |Add Cygwin's bin to PATH; Defaults to: (\cygwin64\bin)|
+|`mkdir %GOPATH%\src\github.com\ipfs`<br/>`cd %GOPATH%\src\github.com\ipfs`<br/>`git clone https://github.com/ipfs/go-ipfs.git`       |Fetch / Update `go-ipfs` source|
+|`cd %GOPATH%\src\github.com\ipfs\go-ipfs` |Change to `go-ipfs` directory|
+|`make install`                            |Build and install to `%GOPATH%\bin\ipfs.exe`|
+|`%GOPATH%\bin\ipfs.exe version --all`     |Test the built binary|
 
-## Manual Way: build go-ipfs
-
-To actually build go-ipfs, first go to the cmd/ipfs directory:
+You can check that the ipfs output versions match with `go version` and `git rev-parse --short HEAD`, if `ipfs.exe` executes and everything matches, then building was successful.
 
+**Tip**: You can permanently append the Cygwin bin to your `PATH` so that on subsequent builds you can simply run
 ```
-cd cmd\ipfs
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
 ```
-
-Then get the current Git commit:
-
+This can be acheived with the `setx` command
 ```
-git rev-parse --short HEAD
+SETX PATH %PATH%;\cygwin64\bin
 ```
 
-It will output a small number of hex characters that you must pass to
-the actual build command (replace XXXXXXX with these characters):
+## Minimal
+While it's possible to build `go-ipfs` with `go` alone, we'll be using `git` and `gx` for practical source management.  
+You can use whichever version of `git` you wish but the Windows builds at <https://git-scm.com> are recommended. `git` must be in your `PATH` for `go get` to recognize and use it.
 
+### `gx`
+`gx` and its go-hook `gx-go` can be built by us.  
+Alternativley you may use prebuilt binaries ([gx](https://dist.ipfs.io/#gx), [gx-go](https://dist.ipfs.io/#gx-go)) by placing them in your `PATH` and skipping this section.
```

Comment:
Flipping this sentence with the previous one places the text about building from source next to the instructions for doing so. That can help clarify exactly what the code block below is supposed to be. e.g:

> You can install prebuilt binaries for gx and gx-go and skip this section. Otherwise, you can install and build them from source by running the following: